libraries/BluetoothSerial/src/BluetoothSerial.cpp

```diff
@@ -1183,4 +1181,18 @@ std::map<int, std::string> BluetoothSerial::getChannels(const BTAddress &remoteA
     return sdpRecords;
 }
 
+void BluetoothSerial::getBtAddress(uint8_t *mac){
+   const uint8_t *dev_mac = esp_bt_dev_get_address();
+   memcpy(mac, dev_mac, ESP_BD_ADDR_LEN);
+}
+
+BTAddress BluetoothSerial::getBtAddressObject(){
+  uint8_t mac_arr[ESP_BD_ADDR_LEN];
+  getBtAddress(mac_arr);
+  return BTAddress(mac_arr);
+}
+
+std::string BluetoothSerial::getBtAddressString(){
+  return getBtAddressObject().toString(true);
+}
```

Comment:
I have checked that the formatting style isn't the same in whole file, but can you align the spaces as is in the function above?